CMakeLists.txt

```diff
@@ -914,9 +920,12 @@ elseif(DRAGONFLY)
   set(EXTRA_LIBRARIES execinfo ${COMPAT})
 elseif(CMAKE_SYSTEM_NAME MATCHES "(SunOS|Solaris)")
   set(EXTRA_LIBRARIES socket nsl resolv)
-elseif(NOT MSVC AND NOT DEPENDS)
-  find_library(RT rt)
-  set(EXTRA_LIBRARIES ${RT})
+elseif(NOT MSVC)
+  CHECK_LIBRARY_EXISTS(rt clock_gettime "time.h" NEED_LIBRT)
+  if(NOT NEED_LIBRT)
+    find_library(RT rt)
+    set(EXTRA_LIBRARIES ${RT})
+  endif()
```

Comment:
That looks wrong.  Check if librt exist, and only set EXTRA_LIBRARIES if not found, but found in another way ?

Reply:
Removed.